pages/profile/content/index.js

```diff
@@ -41,17 +41,17 @@ module.exports = {
   },
   errors: {
     firstName: {
-      required: 'First name is required'
+      required: 'You need to enter a first name.'
     },
     lastName: {
-      required: 'Last name is required'
+      required: 'You need to enter a last name.'
     },
     email: {
-      required: 'Email address is required',
-      match: 'Invalid email address'
+      required: 'You need to enter an email address for this person.',
+      match: 'This email address does not exist.'
```

Comment:
Sorry to nitpick even harder, but what will actually be happening here is that they've entered something which isn't even an email address so I'm not sure "This email address..." makes sense.

Totally happy to merge if you say it's fine btw.